### DISCUSSION TITLE
Log destination of /tpr

### DISCUSSION THREAD

**Discussion by Bobcat00** [EssentialsX/Essentials#4613] [Ideas and Feature Suggestions] [answered]
For server monitoring and debugging purposes, it can be helpful to know where players end up after they do /tpr.

The log message should look something like:
`<player> randomly teleported to <world> <X> <Y> <Z>.`

I suppose the same message could be sent to anyone with the permission essentials.tpr.notify.

**Comment by Bobcat00** [comment]
Thinking about this further, I don't think admins would be interested in in-game notifications of /tpr destinations. I think sending to the log file would be sufficient.

**Reply by JRoy** [reply]
> Thinking about this further, I don't think admins would be interested in in-game notifications of /tpr destinations. I think sending to the log file would be sufficient.

I can do both, one activated by putting tpr in socialspy and the other by a option in tpr.yml

**Comment by JRoy** [comment, marked as answer]
Will be added in https://github.com/EssentialsX/Essentials/pull/4617
